scanpy/plotting/_anndata.py

```diff
@@ -576,15 +577,29 @@ def ranking(
                            top=1-(n_rows-1)*bottom-0.1/n_rows)
     for iscore, score in enumerate(scores.T):
         pl.subplot(gs[iscore])
-        indices = np.argsort(score)[::-1][:n_points+1]
+        if not include_lowest:
+            indices = np.argsort(score)[::-1][:n_points+1]
+        else:
+            sorted_scores = np.argsort(score)[::-1]
```

Comment:
Please call this `score_order` or so, move it before the conditional, and use it in both branches.